src/App/App.js

```diff
@@ -19,18 +20,19 @@ function App() {
   const getMovies = () => {
     getAllMovies().then((movieData) => {
       setMovies(movieData.movies);
-    });
+    })
+    .catch(() => setError("Oops...looks like R A N C I D tomatillos rotted..."));
   };
 
   const viewMovieBlowUp = (id) => {
```

Comment:
I like that you've declared this function in App then are passing it as a prop to be invoked elsewhere.  I'm curious if this approach will need to change once you implement router.